src/python/pants/backend/project_info/filedeps_test.py

```diff
@@ -31,36 +24,19 @@ def rules(cls):
 
     @classmethod
     def target_types(cls):
-        return [GenericTarget, Resources, PythonLibrary]
+        return [MockTarget, ProtobufLibrary]
 
-    @classmethod
-    def alias_groups(cls) -> BuildFileAliases:
-        return BuildFileAliases(
```

Comment:
There was no need to have all these different types registered. Per the Target API, the only fields we care about are `Sources` and `Dependencies`. By definition, a `PythonLibrary` target will be treated no differently than a `Resources` target etc.